agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -581,7 +582,8 @@ func TestFluentdLogTag(t *testing.T) {
 	RequireDockerVersion(t, ">=1.8.0")
 	RequireDockerVersion(t, "<1.12.0")
 
-	if runtime.GOOS == "arm64" {
+	// Skipping the test for arm as they do not have official support for Arm images
+	if runtime.GOARCH == "arm64" {
 		t.Skip()
```

Comment:
Mind adding a string to the output with `t.Skip("unsupported, no image for arm64")` for posterity?